### DISCUSSION TITLE
How to store object other than string in redis?

### DISCUSSION THREAD

**Discussion by mmt4417** [xetorthio/jedis#3258] [Q&A]
How to store object other than string in redis?

Let's say I have a class object which needs to be stored and converted back when get from redis.

**Comment by sazzad16** [comment]
Choose a Java serialization/deserialization library. Serialize your object to String or byte array using the library and save in Redis. And, get String or byte array from Redis and deserialize to object using that library.

Currently there are no _automatic_ way within Jedis for Redis core data structures (i.e. string, hash, etc).

But somethings are available in methods for RedisJSON module commands. Check jsonSet, jsonGet method variants in UnifiedJedis and its sub-classes.
